core/src/main/scala/caliban/Rendering.scala

```diff
@@ -1,259 +0,0 @@
-package caliban
-
-import caliban.Value._
-import caliban.introspection.adt._
-import caliban.introspection.adt.__TypeKind._
-import caliban.parsing.adt.Directive
-
-object Rendering {
```

Comment:
I'm thinking whether we want to keep this object for binary compatibility, annotate it as deprecated and have its public methods and proxy to `rendering.Renderer` instead? Or is is not possible to implement all the methods?

Reply:
Only a few of them were public so yeah would be nice to keep them as deprecated, if possible

Reply:
Fixed!